### DISCUSSION TITLE
GitHub Projects feature

### DISCUSSION THREAD

**Discussion by TSampley** [sinaptik-ai/pandas-ai#72] [General]
GitHub's built-in Projects feature is immensely helpful for organizing issues. Would we be able to turn it on even if just to give an organized overview of issues?

**Comment by gventuri** [comment]
Hey @TSampley, totally agree. As far as I know it should be already tuned on. How do you suggest to handle it?
Personally, I imagine projects as bigger than issues. How would you approach to projects?

**Reply by TSampley** [reply]
If you click on the Projects tab, it says "No Open Projects". You'll need to create a project and then associate it with this repo.

**Reply by gventuri** [reply]
@TSampley just added one, check it out!
